src/test/ide.test.ts

```diff
@@ -322,6 +322,40 @@ test(`we jump to the scripts section for a vanilla script`, async ({rig}) => {
   });
 });
 
+test('jump to definition from object style dependency', async ({rig}) => {
+  const ide = new IdeAnalyzer();
+  await assertDefinition(ide, {
+    path: rig.resolve('package.json'),
+    contentsWithPipe: JSON.stringify(
+      {
+        scripts: {
+          a: 'wireit',
+          b: 'echo',
+        },
+        wireit: {
+          a: {
+            dependencies: [{script: '|b'}],
+          },
+        },
+      },
+      null,
+      2
+    ),
+    expected: {
+      target: `
+    "b": "echo"
+    ~~~~~~~~~~~`,
+      targetSelection: `
+    "b": "echo"
+    ~~~`,
+      // TODO(aomarks) The ~~~ is 2 spaces ahead of where it should be.
+      originSelection: `
+          "script": "b"
```

Comment:
Oh, I bet I know what this is. The comparison that we use ignores leading and trailing whitespace. It probably should just ignore leading newlines. I bet if you add a couple of spaces to this line it will fix it

Reply:
Oh yeah that was it, done.